state/migration_export.go

```diff
@@ -856,24 +856,11 @@ func (e *exporter) addApplication(ctx addApplicationContext) error {
 			endpoints[k] = ep.Name
 		}
 
-		userMap, err := e.st.GetOfferUsers(offer.OfferUUID)
-		if err != nil {
-			return errors.Annotatef(err, "ACL for offer %s", offer.OfferName)
-		}
-
-		var acl map[string]string
-		if len(userMap) != 0 {
-			acl = make(map[string]string, len(userMap))
-			for user, access := range userMap {
-				acl[user] = accessToString(access)
-			}
-		}
-
 		_ = exApplication.AddOffer(description.ApplicationOfferArgs{
 			OfferUUID:              offer.OfferUUID,
 			OfferName:              offer.OfferName,
 			Endpoints:              endpoints,
-			ACL:                    acl,
+			ACL:                    nil,
```

Comment:
Thats correct, I have recorded it [here on Jira](https://warthogs.atlassian.net/jira/software/c/projects/JUJU/issues/JUJU-6577?jql=project%20%3D%20%22JUJU%22%20AND%20text%20~%20%22offers%22%20ORDER%20BY%20created%20DESC).

I've updated the comment there to make it clear that it will result in offers being unconsumable after migration.